src/Yoti.Auth/DigitalIdentity/SharedReceiptResponse.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Yoti.Auth.Profile;
+using Yoti.Auth.Share;
+namespace Yoti.Auth.DigitalIdentity
+{
+    public class SharedReceiptResponse
+    {
+        public string ID { get; set; }
+        public string SessionID { get; set; }
+        public string RememberMeID { get; set; }
+        public string ParentRememberMeID { get; set; }
+        public string Timestamp { get; set; }
+        public string Error { get; set; }
```

Comment:
is the `errorReason` not included because it is going to be added in another PR?

Reply:
have u got another ticket for that yet?